### PR TITLE
remove duplicate description in network.md, fix #1817

### LINKED ISSUE #1817: [Bug Report]: network模块的描述中有重复内容
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:
下面的参数描述内容重复了。
image

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 11dc227717ca040fdd281c704fddeb8eab7e906f

**Comment by skylarCai** [comment]
auto build success,number:220

**Comment by skylarCai** [comment]
autotest fail,number:1587

**Review by wzhiyuan-nit — APPROVED** [review]

File: haas_lib_bundles/python/docs/haas_extended_api/network.md [hunk 1]
```diff
@@ -110,13 +113,6 @@ wlan = network.WLAN(network.STA_IF)
 |ssid|string|是|ssid名称|
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/haas_extended_api/network.md:113]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**